content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+      alt=""
+    />
+    <Caption>Select panel centered with a darker backdrop.</Caption>
+  </div>
+</Box>
+
+## Best practices
+
+### Be predictable
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Select panel with a back button, close button, Heading, subheading, five multiple choice options with three of them selected, and apply and cancel buttons."
+      src="https://user-images.githubusercontent.com/40274682/228386683-20e7ee4e-0d1b-4b6c-9dd1-eddd277b5512.png"
+    />
+    <Caption>If there are more than single options in a menu, have a user confirm their choices.</Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt=""
+      src="https://user-images.githubusercontent.com/40274682/228386685-4699374e-5f98-4dd4-bab8-f488ad0fff8a.png"
+    />
+    <Caption>
+      Never force users to submit their multi-select options by closing the dialog or auto-updating their choices upon
+      selection.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+### Click-to-dismiss behavior
+
+<DoDontContainer>
+  <Do>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a clicking action on the apply button."
+      src="https://user-images.githubusercontent.com/40274682/228388751-12c1f46e-aee7-4e03-8899-b273d31f7d1b.png"
+    />
+    <Caption>
+      Guide users to save or cancel their selections to dismiss Select panel to prevent confusion and data loss.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      width="464"
+      alt="Mouse cursor indicating a click action outside of the dialog."
+      src="https://user-images.githubusercontent.com/40274682/228388749-445619e6-91fa-4f9f-b300-6e1fe21c62a5.png"
+    />
+    <Caption>
+      Don’t allow a click-outside-to-dismiss behavior for Select panel. Users may be used to closing and submitting the
+      information. Doing so would result in a loss of their changes.{' '}
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Behavior
+
+### Opening
+
+<img
+  width="960"
+  alt="Mouse cursor hovering over an icon button and a focused icon button with the space and return keys visually below the icon button."
+  src="https://user-images.githubusercontent.com/40274682/228386681-e75b3bcd-ac0d-4e4a-afda-2a8dfc3a13a6.png"
+/>
+<Caption>
+  A Select panel trigger should be a semantic button. Mouse users can click and keyboard users can open with space or
+  enter.
+</Caption>
+
+<img
+  width="960"
+  alt="First select panel has focus applied to the search input with a blue 2px border. Second SelectPanel has no visible focus ring as focus is brought to the H1 of the dialog."
+  src="https://user-images.githubusercontent.com/40274682/228386679-02d288f4-7e1c-4a77-a654-03def368e266.png"
+/>
+<Caption>
+  If Select panel has an input, focus should be placed within the input. If not, focus should be placed in the next
+  logical place. That next logical place might just be the H1 of the dialog.
```

Comment:
This is a small nit but browsers have an algorithm for moving focus to new floating UI (dialog, popover) and they're unlikely to move it to an `h1`. We might want to adjust this to fall in-line with what the browser is more likely to do: focus on the `autofocus` item, or failing that first item that's a link, form control, or the first item with a non-negative `tabindex`.

Reply:
Yeah I'll revise this. In `SelectPanel`s case, we'll always have interactive elements as the main part of the dialog, so focus will go to the first interactive element. This is more for dialogs in general.